disco/extensions/upgrade_simulation/upgrades/common_functions.py

```diff
@@ -749,4 +1076,103 @@ def compare_dict(old, new):
                 field_list.append(sub_field)
         if change_flag:
             change[key] = field_list
-    return change
\ No newline at end of file
+    return change
+
+
+def create_timepoint_multipliers_dict(timepoint_multipliers):
+    """Creates a dictionary with new load rating, for every property and multiplier.
+    Currently, it only does this for loads. But can be modified to accommodate other elements like PV as well.
+    In raw_dict, value can be accessed as follows:
+    value = raw_dict[property_name][object_name][multiplier_name]
+    
+    In reformatted_dict (which is returned from this function), value can be accessed as follows:
+    value = raw_dict[object_name][property_name][multiplier_name]
+    This value will need to be assigned to the object and run.
+    This hasnt been used yet.
+    
+    Returns
+    -------
+    dict
+    """
+    field = "load_multipliers"
+    if field == "load_multipliers":

```

Comment:
Redundant

Reply:
True, I added a few more lines, to loop over dict keys and raise exception if key other than this is present.
That was my main intent.